src/displayapp/screens/PassKey.cpp

```diff
@@ -0,0 +1,31 @@
+#include "PassKey.h"
+#include "../DisplayApp.h"
```

Comment:
Maybe we can avoid relative imports in new code?

Reply:
Agreed